### DISCUSSION TITLE
iTunes (PC)->shairport-sync(pi)->Alexa speaker

### DISCUSSION THREAD

**Discussion by arnieSkyNet** [mikebrady/shairport-sync#1570] [General]
how can I configure shairport-sync on my raspberry pi to play everything sent to it from my iTunes (PC) to shairport-sync to the Bluetooth output of the raspberry pi which is connected to my Alexa via Bluetooth (I can play audio wave’s using aplay) 

Just wondering what settings on shairport to edit and change?

**Comment by mikebrady** [comment]
This is a really difficult thing to do (because Bluetooth) and some have tried to do it before. Web search is your friend here, and maybe look through some of the older (closed) issues.
